lib/raven/context_filter.rb

```diff
@@ -0,0 +1,40 @@
+module Raven
+  module ContextFilter
```

Comment:
think we can move it under `lib/utils/context_filter` and name it `Raven::Utils::ContextFilter`